app/code/core/Mage/Media/Model/File/Image.php

```diff
@@ -111,7 +126,7 @@ public function getImage(Mage_Media_Model_Image $object)
      * Create tmp image resource for operations
      *
      * @param Mage_Media_Model_Image $object
-     * @return $this
+     * @return false|resource
```

Comment:
Shouldn't it be `@return $this`?

Reply:
Ne, it returns `        $resource = imagecreatetruecolor($object->getDestanationDimensions()->getWidth(), $object->getDestanationDimensions()->getHeight());
        return $resource;`

Reply:
Yes, indeed. Thanks!